lib/mobius/cog.ex

```diff
@@ -81,28 +81,26 @@ defmodule Mobius.Cog do
       alias Mobius.Actions.Events
       alias Mobius.Services.Bot
 
-      @computed_commands Enum.reverse(@commands)
+      @computed_commands Command.preprocess_commands(@commands)
+
+      defp __commands__, do: @computed_commands
 
       listen :message_create, message do
-        case Command.execute_command(@computed_commands, Bot.get_global_prefix!(), message) do
+        case Command.execute_command(__commands__(), Bot.get_global_prefix!(), message) do
           {:ok, _} ->
             :ok
 
-          {:too_few_args, command, received} ->
+          {:too_few_args, arities, received} ->
             Logger.info(
-              "Wrong number of arguments for command \"#{command.name}\". Expected #{
-                Command.arg_count(command)
+              "Wrong number of arguments. Expected one of #{
+                Enum.join(Enum.map(arities, &Integer.to_string/1), ", ")
```

Comment:
```suggestion
                arities |> Enum.map(&Integer.to_string/1) |> Enum.join(", ")
```

Don't know how you feel about single line pipes but I think this reads much better.

Reply:
Sure, I thought it was better since it's inside a string interpolation, but that works too